File: src/main/java/attendance/model/Attendance.java
package attendance.model;

import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.time.Instant;

@Data
@Entity
public class Attendance {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private Long studentId;

    private Instant date;

    private int roll;

    private String status;

    private String department;

    private String semester;

    private String subject;
}

File: src/main/java/attendance/model/Attendance.kt
package attendance.model

import com.fasterxml.jackson.annotation.JsonFormat
import lombok.Data

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id
import java.time.Instant

@Entity
data class Attendance(
        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        var id: Long,
        var studentId: Long,
        @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "UTC+6")
        var date: Instant,
        var roll: Int,
        var status: String,
        var department: String,
        var semester: String,
        var subject: String
)
